### PR TITLE
Added assertBinary method to RestResponse class.

### PR DESCRIPTION
**Pull request by antonini**
Added assertBinary method to RestResponse class.
This was requested on issue #73.

### CODE REVIEW

**Comment by dmarkov** [comment]
@antonini Thanks, I will find someone to review this PR soon

**Comment by dmarkov** [comment]
@ggajos it's yours, please review

**Comment by antonini** [comment]
Fixed check style problems.

File: src/test/java/com/jcabi/http/request/FakeRequestTest.java [hunk 1]
```diff
@@ -49,6 +49,21 @@
...
+
+    /**
+     * Body message: how are you?
+     */
+    private static final String BODY_HOW_ARE_YOU = "how are you?";
+
+    /**
+     * Path of hello all request.
+     */
+    private static final String PATH_HELLO_ALL = "/helloall";
```
Comment (ggajos) [inline src/test/java/com/jcabi/http/request/FakeRequestTest.java]: @antonini I think we should not coupling tests together, we can put any different string here, right?

File: src/test/java/com/jcabi/http/request/FakeRequestTest.java [hunk 2]
```diff
@@ -68,6 +83,26 @@ public void sendsHttpRequestAndProcessesHttpResponse() throws Exception {
...
+     * @throws Exception If something goes wrong inside
+     */
+    @Test
+    public void sendsHttpRequestAndProcessesHttpBinaryResponse()
+        throws Exception {
+        new FakeRequest()
+            .withStatus(HttpURLConnection.HTTP_OK)
+            .withReason(REASON_OK)
+            .withHeader(HttpHeaders.CONTENT_TYPE, MediaType.TEXT_PLAIN)
+            .withBody(BODY_HOW_ARE_YOU.getBytes())
```
Comment (ggajos) [inline src/test/java/com/jcabi/http/request/FakeRequestTest.java]: @antonini if many lines are the same as in other test case maybe we can do some helper class to make it more DRY?

File: src/test/java/com/jcabi/http/request/FakeRequestTest.java [hunk 3]
```diff
@@ -68,6 +83,26 @@ public void sendsHttpRequestAndProcessesHttpResponse() throws Exception {
     }
 
     /**
+     * FakeRequest can fetch a fake response.
```
Comment (ggajos) [inline src/test/java/com/jcabi/http/request/FakeRequestTest.java]: @antonini name of test is different than description

**Comment by ggajos** [comment]
@antonini few comments

File: src/test/java/com/jcabi/http/request/FakeRequestTest.java (hunk 1 above)
Comment (antonini) [inline src/test/java/com/jcabi/http/request/FakeRequestTest.java]: Agree. Working on that.

File: src/test/java/com/jcabi/http/request/FakeRequestTest.java (hunk 2 above)
Comment (antonini) [inline src/test/java/com/jcabi/http/request/FakeRequestTest.java]: Agree. Changed to an external method.

File: src/test/java/com/jcabi/http/request/FakeRequestTest.java (hunk 3 above)
Comment (antonini) [inline src/test/java/com/jcabi/http/request/FakeRequestTest.java]: Agree. Improved the javadoc.

**Comment by antonini** [comment]
@ggajos All changes was done, can you check that again please?

**Comment by ggajos** [comment]
> Fixed check style problems.
@antonini please remember to address comments directly

**Comment by antonini** [comment]
@ggajos Ok. I'll try to do better comments.

**Comment by ggajos** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by dmarkov** [comment]
@ggajos thanks, paid, **18 mins** to your account, payment ID is `AP-0UH46051997517221`, task took 67 hours and 33 mins to complete

extra minutes for review comments (c=3)

+18 added to your rating, at the moment it is: [+2478](http://www.netbout.com/b/35094?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy now pls

**Pull request merged**